### DISCUSSION TITLE
Revealjs feature request: line number offset

### DISCUSSION THREAD

**Discussion by andrewpbray** [quarto-dev/quarto-cli#1451] [Feature Requests]
Version 4.2.0 of revealjs implemented a feature that allows you to specify the line number on which a code chunk will start. I don't see it as a must-have feature, but it'd be helpful when displaying more than one chunk of code from the same file.

Description of feature:
[https://revealjs.com/code/#line-number-offset-4.2.0](https://revealjs.com/code/#line-number-offset-4.2.0)

Changelog: 
[https://github.com/hakimel/reveal.js/releases/tag/4.2.0](https://github.com/hakimel/reveal.js/releases/tag/4.2.0)

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/1454
